spt/features/noclip_nofix.cpp

```diff
@@ -92,39 +92,42 @@ void NoclipNofixFeature::LoadFeature()
 			if (memcmp((void*)(bytes + i), (void*)inscCompare, 4))
 				continue;
 
-			ptrJumpInsc = (uintptr_t)(bytes + i + 2);
-			memcpy((void*)origJumpBytes, (void*)(ptrJumpInsc), 6);
+			INIT_BYTE_REPLACE(Jump, (uintptr_t)(bytes + i + 2));
 
 			/*
 			* When we replace the instruction with a JMP, we replace the last byte with NOP,
 			* meaning the starting point of our jump is shifted up by 1 to that byte.
 			*/
 			jmpDistance = jmpOff + 1;
+			goto success;
 		}
-	}
 
+	}
+	return;
+	
+	success:;
 	InitConcommandBase(y_spt_noclip_nofix);
```

Comment:
you could just put the InitConcommandBase in the inner loop here and get rid of the goto